ms-common-auth/src/main/java/net/trajano/ms/authz/TokenResource.java

```diff
@@ -18,6 +18,8 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.UriBuilderException;
 
+import net.trajano.ms.core.ErrorResponses;
+import net.trajano.ms.core.Qualifiers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.trajano.ms.core.Qualifiers'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)